packages/dnb-eufemia/src/components/accordion/AccordionHeader.tsx

```diff
@@ -97,15 +98,27 @@ export type AccordionHeaderIconProps = {
   icon?: AccordionHeaderIconIcon
   size?: IconSize
   expanded?: boolean
+  iconRight?: boolean
 }
 
 function AccordionHeaderIcon({
   icon,
   expanded,
   size = 'medium',
+  iconRight
 }: AccordionHeaderIconProps) {
+  const theme = useTheme()
+  let animateIcon = true;  
+  if(!icon && theme && theme.name === 'sbanken' ) {
+    animateIcon = false;
+    icon = {
+      expanded:'subtract-medium',
```

Comment:
```suggestion
      expanded: 'subtract-medium',
```
Missing prettier formatting?

Reply:
apparently prettier crashes on this document because it doesn't support then `satisfies` typescript keyword. I've upgraded to version prettier `^2.8.0` where they add support.